builtin/provisioners/chef/resource_provisioner.go

```diff
@@ -252,6 +256,23 @@ func Provisioner() terraform.ResourceProvisioner {
 				Type:     schema.TypeString,
 				Optional: true,
 			},
+			// Same defaults as Test-Kitchen
+			// https://github.com/test-kitchen/test-kitchen/blob/e5998e0dd1aa42601c55659da78f9b112ff9f8ee/lib/kitchen/provisioner/base.rb#L36-38
```

Comment:
These comments should probably not be here, but instead they options should be added and described in the docs (which are also in the repo in the `website` folder).